app_dart/lib/src/model/appengine/task.dart

```diff
@@ -178,6 +179,13 @@ class Task extends Model<int> {
   @JsonKey(name: 'Flaky')
   bool isFlaky;
```

Comment:
Are you able to update this doc to help differentiate it from the new property?

Reply:
Updated the doc to reflect latest .ci.yaml, also to distinguish from the new property.